tests/unit/containers/edit-profile/profile-tab/profile-tab-form/ProfileTabForm.spec.jsx

```diff
@@ -53,6 +56,36 @@ describe('ProfileTabForm', () => {
     expect(languageField.value).toBe(newLanguageValue)
   })
 
+  it('should display the initial native language value', () => {
+    const languageField = screen.getByLabelText(
+      'becomeTutor.languages.autocompleteLabel'
+    )
+
+    expect(languageField.value).toBe('English')
+  })
+
+  it('should open the dropdown with all language options', async () => {
+    const languageField = screen.getByLabelText(
+      'becomeTutor.languages.autocompleteLabel'
+    )
+    fireEvent.mouseDown(languageField)
+
+    const options = [
+      'English',
+      'Ukrainian',
+      'Polish',
+      'German',
+      'French',
+      'Spanish',
+      'Arabic'
+    ]
+
+    for (const lang of options) {
+      const option = await within(document.body).findByText(lang)
```

Comment:
```suggestion
      const option = await screen.findByText(lang)
```
Will this work without using `within(document.body)`?

Reply:
Yeah, you are right, fixed that